packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,331 @@
+
+## Standard cards
+PatternFly supports several common card layouts to help convey information in a consistent way. This is not an exhaustive list, but before designing your cards, you should see if the information you are trying to communicate fits into any of these common patterns.
+
+### Aggregate status card
+Aggregate status cards show a total number of objects and an aggregated status for them. For example, it can be used to provide users with a quick count of the number of nodes making up a large distributed network and identify the numbers that are down, that need maintenance, etc. There would also typically be a link to examine more details.
+
+<img src="./img/aggregate-status-cards.png" alt="aggregate status card examples" width="542"/>
+
+1. **Card title:** Reports the total count of objects being monitored.
+
+2. **Normal status:** Reflects that all is running normally, with no problems.
+
+3. **Exception status (count only):** Counts the number of items that are reporting exceptions (errors, warnings, or other conditions). Only include non-zero items. You may optionally link the count to more details.
+
+4. **Counts with annotation (optional):** Include a label and description if more information may be required to clarify the reported status.
+
+### Trend card
+Trend cards are used to display the change in some metric over time. It should be possible to see the current value and prior values over some given time period.
+
+<img src="./img/trend-cards.png" alt="trend card examples" width="372"/>
+
+1. **Current value:** Use the title of the card to reflect the current value.
+
+2. **Filter (optional):** Place a filter in the upper right corner of the card when you want to filter data in the chart. Most often this will be used to change the time range of the chart, e.g. “Last 24 hours,” “Last 7 days,” “Last 30 days,” etc.
+
+3. **Trend chart:** [Sparkline charts](/charts/sparkline-chart) are most often used to display trend data.
+
+4. **Actions (optional):** If your card is to include actions include them as [link-style buttons](/components/button#variations) in the footer of your card.
+
+Regular trend cards arrange the current value and trend chart in a stacked layout. You can use the short trend card layout to save vertical space by placing the current value and the trend chart side-by-side. You may also want to monitor trends for several metrics on the same card.
+
+
+If you’re displaying multiple trend metrics on a card, stack trend charts with clear naming to indicate what each chart measures and add a title at the top to communicate what’s being displayed on this card.
+
+### Utilization card
+A utilization card reports the current value of a single metric or multiple metrics. Bar or donut charts are most often used to report utilization because values are proportional (i.e. percentage of a whole).
+
+<img src="./img/utilization-cards.png" alt="utilization card examples" width="907"/>
```

Comment:
Noticed that in this picture, the pink headers are above the pictures, whereas in the above images, the pink headings are below the images. Would probably try and keep that consistent!